frontend/src/utils/date-utils.ts

```diff
@@ -100,22 +114,46 @@ export const dateStrToUpperCase = (dateStr: string): string => {
 };
 
 export const getDateFormat = (
```

Comment:
might be worth renaming to `getFormattedDate`, wdyt?

Reply:
I agree, I made the same comment on the other PR about testing, before seeing your comment here.

Reply:
I pushed the change yesterday :)